app/views/account_dashboard_statuses.scala.html

```diff
@@ -0,0 +1,127 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import utils.Dates
+
+@import config.FrontendAppConfig
+@import java.time.Instant
+@import java.time.temporal.ChronoUnit
+@import components.pagination
+
+
+@(appConfig: FrontendAppConfig,  cases: Paged[Case])(implicit request: Request[_], messages: Messages)
+
+@paginate(containerClass: String, idPrefix: String) = {
+@pagination(
+id = s"$idPrefix-applications-pagination",
+singularName = messages("case.applications.pagination.singular"),
+pluralName = messages("case.applications.pagination.plural"),
+pager = cases,
+containerClass = Some(containerClass),
+onChange = page => controllers.routes.IndexController.getApplicationsAndRulings(page)
+)
+}
+
+@main_template(
+title = messages("index.title"),
+appConfig = appConfig,
+contentClass = Some("column-full"),
+bodyClasses = None) {
+
+
+<h1 class="heading-xlarge">
+ <span class="heading-secondary">@messages("application.dashboard.status.title")</span>
+ @messages("index.heading")
+</h1>
```

Comment:
Could we potentially use the `heading` component for this now that it has the `captionMsg` parameter to add a secondary heading?